README.md

```diff
@@ -18,6 +18,39 @@ An interactive label running on a small form factor PC connected to a touchscree
 * Dell Optiplex 3070 Micro i3 small form factor PC
 * [A set of lens reader hardware](https://github.com/ACMILabs/lens-reader) if lens reader integration is needed
 
+## Generate background layout files
+
+See the README in the [background-image-tools](background-image-tools) folder
+
+## Run a development server with docker
+
+* Run `cp config.tmpl.env config.env`
+* Run `cd development`
+* Run `docker-compose up --build`
+* Open a browser and visit: http://localhost:8081
+
+## Run Javascript tests with docker
+
+* Run `cd testing`
+* Run `docker-compose up --build`
+* In another Terminal run `docker exec -it javascripttests make linttestjs`
+
+## Run Python tests with docker
+
+* Run `cd development`
+* Run `docker-compose up --build`
+* In another Terminal run `docker exec -it labelinteractive make linttest`
+
```

Comment:
I got a lot of linting errors when I ran this, on the following modules:
LICENSE, bg_img_tools.mm_process, bg_img_tools.mi_process, bg_img_tools.mw_process, bg_img_tools.fsf_process, tests.tests

Reply:
I forgot to rename the linting ignores after changing the directory to a more human readable name.